tajo-core-tests/src/test/resources/results/TestTajoCli/testHelpSessionVars.result

```diff
@@ -42,4 +42,5 @@ Available Session Variables:
 \set ARITHABORT [true or false] - If true, a running query will be terminated when an overflow or divide-by-zero occurs.
 \set FETCH_ROWNUM [int value] - Sets the number of rows at a time from Master
 \set BLOCK_ON_RESULT [true or false] - Whether to block result set on query execution
+\set RESULT_COMPRESS [true or false] - Enable resultSet compression
```

Comment:
Thanks for your detailed review
"useCompression=true" looks great to me. also I’m going to change to COMPRESSED_RESULT_TRENSFER for session variable